src/daemon/instance_settings_handler.cpp

```diff
@@ -166,14 +166,19 @@ void update_bridged(const QString& key,
     // This is the user parameter, true or false.
     auto bridged = mp::BoolSettingSpec{key, "false"}.interpret(val) == "true";
 
-    if (!bridged && is_bridged(instance_name))
+    if (is_bridged(instance_name))
     {
-        throw mp::InvalidSettingException{key, val, "Bridged interface cannot be removed"};
+        if (!bridged)
+        {
+            throw mp::InvalidSettingException{key, val, "Bridged interface cannot be removed"};
+        }
```

Comment:
I think it deserves an `info` log saying that the instance is already bridged.

Reply:
Good, done.